core/src/main/java/org/mule/DefaultMuleMessage.java

```diff
@@ -1607,16 +1565,6 @@ private MuleMessage copyToInbound(DefaultMuleMessage currentMessage, Object payl
         return newMessage;
     }
 
-    void setSessionProperties(Map<String, TypedValue> sessionProperties)
-    {
-        properties.sessionMap = sessionProperties;
-    }
-
-    void setInvocationProperties(Map<String, TypedValue> invocationProperties)
```

Comment:
These methods were used for the event to inject whole map into the message when setMessage() is called, this is no longer needed.